pkg/kustomize/kustomize.go

```diff
@@ -324,7 +333,7 @@ func KustomizationFor(ctx *quaycontext.QuayRegistryContext, quay *v1.QuayRegistr
 			componentPaths = append(componentPaths, filepath.Join("..", "components", string(component.Kind)))
 			managedFieldGroups = append(managedFieldGroups, fieldGroupNameFor(component.Kind))
 
-			componentConfigFiles, err := componentConfigFilesFor(component.Kind, quay, quayConfigFiles)
+			componentConfigFiles, err := componentConfigFilesFor(ctx, component.Kind, quay, quayConfigFiles)
```

Comment:
Nit: Maybe a different name here would be a good idea, `ctx` is traditionally always `context.Context`. `qCtx` or `quayCtx` might distinguish it more.

Reply:
Following pre established standards here. I fully agree with you.